src/plantumlv2/pu_manager.py

```diff
@@ -213,7 +201,8 @@ def _filter_packages(
                     filtered_packages_set.add(package)
 
             if isinstance(package_view, dict):
-                filter_path = package_view["packagePath"].replace(".", "/")
+                filter_path = package_view["path"].replace(".", "/")
+                filter_path = filter_path.replace("*", "")
```

Comment:
uff, i don't have the energy to figure out what's happening here. why remove the star in the path? 
i guess you understand it and this is what matter :)

Reply:
Removing the star is just an easy hack to set the filepath to be empty so that all packages are considered as before.

Reply:
so the star does not function like a regex? I was sure it did.

Reply:
Yeah, the star works as a regex, however, it is still parsed as a star, so removing it here allows the reuse of the existing code.